Sources/NYT360CameraController.m

```diff
@@ -10,6 +10,8 @@
 #import "NYT360EulerAngleCalculations.h"
 #import "NYT360CameraPanGestureRecognizer.h"
 
+static const CGPoint kStartingPosition = { 3.14, 0 };
```

Comment:
Thanks for the fix.
Can you rename the constant variable? The `k` is very old school and typically in ObjectiveC constants start with a lower level.

Reply:
yup, fixed